### PR TITLE
QC checks for v13 rnaseq merged files

### PR DESCRIPTION
**Pull request by kgaonkar6**
All merged RNAseq files are in https://cavatica.sbgenomics.com/u/cavatica/pbta/files/5e1c4c37e4b0e4c507ca5ce3/
this includes gene and isoform tpm,fpkm and count files (970 stranded and 58 polya from 1028 v13 RNAseq files)

This markdown helps with QC for new RNAseq rsem.genes (v13) files to old RNAseq rsem files (v11 because v12 had issues)
1. PRINTS: dimensions of new and old datasets
2. CHECKS: gene overlap between new and old datasets
3. CHECKS: Sample overlap with clinical files
4. CHECKS: gene order with newly formed stranded data and old polya datasets
5. If all above checks pass then final "Checks PASSED/Checks FAILED" is printed

Additionally fusion files are QC-ed by :
##### Steps

1. PRINTS: dimensions of new and old datasets
2. CHECKS: sample overlap between new and histology file
3. CHECKS: TBD
4. CHECKS: TBD
5. If all above checks pass then final "Checks PASSED/Checks FAILED" is printed

### CODE REVIEW

**Comment by jharenza** [comment]
@kgaonkar6  - I missed that you went from v11 to v13 - do we know if all v11 files were merged correctly to begin with?

**Comment by kgaonkar6** [comment]
Yes I believe v11 rsem files were correctly merged in terms of samples used (1028 ; 970 stranded and 58 polya after removing mismatched samples etc. ) and gene order number of genes in polya dataset matched stranded dataset. 
Let me know if you would rather QC merged files with a randomly selected rsem file for gene order and number of genes, that should be doable as well.

File: OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+5. If all above checks pass then final "Checks PASSED/Checks FAILED" is printed
+
+
+```{r Read input}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+#load required packages
+suppressPackageStartupMessages(library("readr"))
+suppressPackageStartupMessages(library("tidyverse"))
+suppressPackageStartupMessages(library("reshape2"))
+suppressPackageStartupMessages(library("qdapRegex"))
```
Comment (jharenza) [inline OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd]: add some code to install these packages if not installed

File: OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    label: "Clinical file" 
+    value: data/rerun-v13/2020-01-13-pbta-histologies-v13.tsv
+    input: file
+  oldDataFolder:
+    label: "Input old version of rsem files stranded"
+    value: data/release-v11-20191126/
+    input: file
+  toAddDataFolder:
+    label: "Input new data of rsem files stranded to add"
+    value: data/rerun-v13/
```
Comment (jharenza) [inline OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd]: Add instructions for 1) user to create new files and create a tarball, then 2) download and add the rerun-v13 tarball to a specific folder as you note above

Comment (kgaonkar6) [reply OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd]: I added instruction to git clone the data https://github.com/d3b-center/D3b-codes/blob/b03c62478db8fe7759bed428448a79c7f97f0256/OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13_gene.Rmd#L26 .  For old and new folder to compare I think we can use rmarkdown::render() to overide the params given here by specifying it while running like in the first example here: https://bookdown.org/yihui/rmarkdown/params-knit.html

File: OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd (hunk 1 above)
Comment (kgaonkar6) [reply OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13.Rmd]: added code https://github.com/d3b-center/D3b-codes/blob/b03c62478db8fe7759bed428448a79c7f97f0256/OpenPBTA_V13_release_QC_scripts/QC_RNAseq_v13_gene.Rmd#L42 to install packages if not available

**Comment by jharenza** [comment]
thanks for the updates!

**Pull request merged**